wee_slack.py

```diff
@@ -819,7 +819,7 @@ def typing_notification_cb(signal, sig_type, data):
         if typing_timer + 4 < now:
             current_buffer = w.current_buffer()
             channel = EVENTROUTER.weechat_controller.buffers.get(current_buffer, None)
-            if channel and channel.type != "thread":
+            if channel and hasattr(channel, "type") and channel.type != "thread":
```

Comment:
I'm thinking we can add `type = 'team'` to `SlackTeam` instead of changing this check.

Reply:
Sounds good!